docs/data/routes/docs/nextjs/deploying-to-production/export/en.md

```diff
@@ -3,4 +3,76 @@ name: export
 routeTemplate: ./data/component-templates/article.yml
 title: Using `next export` with the Next.js sample app
 ---
-# Walkthrough: Using `next export` with the Next.js sample app
\ No newline at end of file
+# Walkthrough: Using `next export` with the Next.js sample app
```

Comment:
Just realized we should say something here about the fact that the Sitecore Media Library is not exported, and that a running Content Delivery instance is still needed for the media library.